.github/workflows/semver-checks.yml

```diff
@@ -22,15 +22,15 @@ jobs:
         run: cargo binstall cargo-semver-checks --no-confirm
       - name: "Run semver checker script"
         run: ./contrib/check-semver.sh
-      - name: "Add PR label to breaking changes"
-        uses: actions-ecosystem/action-add-labels@v1
+      - name: Save PR number
         if: ${{ hashFiles('semver-break') != '' }}
-        with:
-          labels: "API break"
-      - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v2
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          echo $PR_NUMBER > ./semver-break
```

Comment:
@Kixunil I cannot read the PR number in the `semver-checks-pr-label.yml` from the previous `semver-checks-pr.yml` workflow.
I've tried a couple of things, most notoriously this:

```yml
    env:
      # hack to get PR number from the workflow_run object
      PR_NUMBER: ${{ github.event.workflow_run.conclusion.pull_requests[0].number }}
```

Reply:
That sucks. But if someone griefs us we'll have an argument for GH to fix that.